### PR TITLE
[MIG] keychain to 11

### PR DESCRIPTION
**Pull request by mart-e**
Move to server-auth
Add base_sparse_field to get Serialized fields
Run 2to3
Bump version number
Remove coding utf-8

_As it is the [hacktoberfest](https://hacktoberfest.digitalocean.com/), it is the occasion to help outside open source projects_

Confession : I haven't properly tested it (yet) as I need other modules extending the module and adding `keychain.account` namespaces.

### CODE REVIEW

**Review by yajo — APPROVED** [review]
Code OK, at first sight, but don't merge until bots are happy.

File: keychain/tests/test_keychain_backend.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def test_keychain_bakend(self):
         backend._inverse_keychain()
         account = backend._get_existing_keychain()
         self.assertEqual(
-            account.data, '{"a": "o", "c": "b"}',
+            account.get_data(), {"a": "o", "c": "b"},
```
Comment (pedrobaeza) [inline keychain/tests/test_keychain_backend.py]: You can use `self.assertDictEqual` instead for not having to worry about the order.

**Comment by mart-e** [comment]
Is there still something to fix? The runbot error seems like a missing library to install.

**Comment by pedrobaeza** [comment]
It complains about `Pygments` library. I don't know for what is that library used, but you can add it on a requirements.txt file on the root folder for having it installed on runbot too.

**Comment by yajo** [comment]
Looks pretty nice now, could you please squash all migration-related commits into one, for merging? Thanks

**Comment by pedrobaeza** [comment]
I'd squash then.

Thank you very much for the contribution, Martin.

**Pull request merged**